plugins/hadoopsecuritymanager-yarn/src/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -79,22 +84,33 @@
   public static final String RM_PRINCIPAL = "yarn.resourcemanager.principal";
 
   public static final String HADOOP_JOB_TRACKER = "mapred.job.tracker";
-  public static final String HADOOP_JOB_TRACKER_2 =
-      "mapreduce.jobtracker.address";
+  public static final String HADOOP_JOB_TRACKER_2 = "mapreduce.jobtracker.address";
   public static final String HADOOP_YARN_RM = "yarn.resourcemanager.address";
 
   private static final String OTHER_NAMENODES_TO_GET_TOKEN = "other_namenodes";
 
+  /**
+   * the settings to be defined by user indicating if there are hcat locations
+   * other than the default one the system should pre-fetch hcat token
+   * from. Note: Multiple thrift uris are supported, use comma
+   * to separate the values, values are case insensitive.
+   * */
+  private static final String EXTRA_HCAT_LOCATION = "extra.hcat.location";
```

Comment:
just a thought. Should we follow azkaban nomenclature like other_namenode ? We can rename it to other.hcat.location ?